netaddr/ip/__init__.py

```diff
@@ -961,11 +964,15 @@ class IPNetwork(BaseIP, IPListMixin):
     .. versionchanged:: 1.1.0
         Removed partial IPv4 address support accidentally left when making 1.0.0 release.
         Use :func:`expand_partial_ipv4_address` if you need this behavior.
+
+    .. versionchanged:: NEXT_NETADDR_VERSION
```

Comment:
Not sure if this is supposed to be an actual number, otherwise lgtm.

Reply:
It's a placeholder used in the project, it's good